test/clj_kondo/unresolved_symbol_test.clj

```diff
@@ -241,8 +250,13 @@
  (:foo x))
 
 " '{:linters {:unresolved-symbol {:exclude [(slingshot.slingshot/try+)]}}})))
-  (is (empty? (lint! "(def ^name.fraser.neil.plaintext.diff_match_patch dmp (diff_match_patch.))"
-                     '{:linters {:unresolved-symbol {:level :error}}}))))
+  (assert-submaps
+   '({:file "<stdin>", :row 1, :col 56, :level :error, :message "Unresolved symbol: diff_match_patch"})
+   (lint! "(def ^name.fraser.neil.plaintext.diff_match_patch dmp (diff_match_patch.))"
```

Comment:
I think this test was added to test that `foo.bar.foo_bar` is detected as a class name despite not having capital letters. Could you leave that in? The constructor call can be removed.

Reply:
See https://github.com/clj-kondo/clj-kondo/issues/1362

Reply:
I saw that and thought that we could lint only the constructor, but makes sense to remove the constructor since it's not what the issue is related to, will do

Reply:
Done! @borkdude